<ui version="4.0" >
 <class>LogWindow</class>
 <widget class="QDialog" name="LogWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Log Window</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTableWidget" name="tableWidget" >
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Source</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>181</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="text" >
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
